### ISSUE TITLE
use

### ISSUE THREAD

**Issue by TodePond** [TodePond/DreamBerd#44]
useUse

**Comment by FoxNerdSaysMoo** [comment]
I agree, for proper code readability all function calls should be denoted as `use myfunc()`. This improves code readability by showing that you are actually using the function, rather than just mentioning it.
